HalfMagicProximity/App.cs

```diff
@@ -20,12 +20,16 @@ static void Main(string[] args)
                 ArtManager artManager = new ArtManager(cardManager.Cards);
 
                 // Generate the front faces of adventures and all split cards with the normal M15 template
-                proximityManager.Run(isSketch:false);
-                artManager.CleanProxies(isSketch: false);
+                proximityManager.Run(CardTemplate.M15);
```

Comment:
Is there a way to make this more automatically extensible?

Reply:
Probably. Don't think it's worth the effort while there are still only three templates